### PR TITLE
add documentation to repo

### LINKED ISSUE #1547: Add website contents to code repo (this)
A simpler documentation model:
https://github.com/blog/2228-simpler-github-pages-publishing

This will simplify issues related to documentation, as well as enable documentation checks as part of CI and build scripts.

### PR DESCRIPTION
**Pull request by nids2001**
Documentation moved to /docs
Closes #1547

### CODE REVIEW

**Comment by dcwhite** [comment]
Cool. I need to make another issue, to coordinate the website repo build script into the main build script.

**Comment by dcwhite** [comment]
@nids2001 Could you investigate and try to fix the CodeFactor warnings? They look reasonable, as far as being easy to fix and probably worth it to clean up the css code. 
https://www.codefactor.io/repository/github/sciinstitute/scirun/pull/2146

File: docs/build.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<link rel="stylesheet" href="css/main.css">
+
+## Platform Notes
+
+**At least C++11 64-bit compiler support is required.**
+
+### Build requirements
+
+#### OS X
+  - Tested on 10.10 - 10.13
```
Comment (jessdtate) [inline docs/build.md:17]: - Tested on 10.12 -10.15

File: docs/build.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  - Apple clang 7.2 or newer
```
Comment (jessdtate) [inline docs/build.md:18]: 9.0.0 or newer

File: docs/build.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  - Qt 4.8 or 5.12
```
Comment (jessdtate) [inline docs/build.md:19]: Qt 5.13

Comment (jessdtate) [reply docs/build.md:19]: also remove all qt 4.8 references

File: docs/build.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+#### Linux
+  - Tested on Ubuntu 14.04 LTS, OpenSUSE Leap 42.1
+  - gcc 4.8
+  - Qt 5.12 or later
+    + [Download](https://download.qt.io/archive/qt/) and run the Qt 5.12 installer.  Make sure to turn off other versions and system configurations to save space and build time.  Add the Qt 5 path to CMake configuration.
+
+#### All Platforms
+  - [CMake](https://cmake.org/) (platform independent configuring system that is used for generating Makefiles, Visual Studio project files, or Xcode project files)
+    + Tested with 2.8 - 3.4
```
Comment (jessdtate) [inline docs/build.md:40]: cmake 3.4 and newer

**Comment by dcwhite** [comment]
25->10 codefactors...good progress!

**Comment by dcwhite** [comment]
Decided to merge this still with a few css errors, then fix those later since it will be easier to see the problems with a smaller diff/number of files.

**Comment by dcwhite** [comment]
@jessdtate Today? I'm eager to see the new http://sciinstitute.github.io/SCIRun/

File: docs/build.md [hunk 5]
```diff
@@ -14,30 +14,27 @@ layout: default_toc
...
   - Visual Studio 2013 & 2017
-  - Qt 5.12 or later
-    + [Download](https://download.qt.io/archive/qt/) and run the Qt 5.12 installer.  Make sure to turn off other versions and system configurations to save space and build time.
+  - Qt 5.13 or later
+    + [Download](https://download.qt.io/archive/qt/) and run the Qt 5.13 installer.  Make sure to turn off other versions and system configurations to save space and build time.
 
 
 #### Linux
   - Tested on Ubuntu 14.04 LTS, OpenSUSE Leap 42.1
   - gcc 4.8
```
Comment (jessdtate) [inline docs/build.md:31]: This line should also be removed at some point

Comment (dcwhite) [reply docs/build.md:31]: We can fix that as part of https://github.com/SCIInstitute/SCIRun/issues/2150

**Pull request merged**